net/dns/config.go

```diff
@@ -78,13 +79,14 @@ func (c Config) hasRoutes() bool {
 }
 
 // hasDefaultIPResolversOnly reports whether the only resolvers in c are
-// DefaultResolvers, and that those resolvers are simple IP addresses.
+// DefaultResolvers, and that those resolvers are simple IP addresses
```

Comment:
I have spent a minute trying to decide if that should be an `and` or an `or`. I don't know. Maybe this function needs a different name? `tailscaleOwnsDNS`?